azure/notebooks/Azure-MNMG-RF.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<p>One does not <em>have</em> to use the following function to allocate workers. There are two ways to allocate the workers:</p><ol><li>If you pass <code>n_workers</code> in AzureVMCluster API, then the workers will be allocated. However, AzureVMClusters will return before the workers are actually up and running. So you will have to call <code>client.wait_for_workers(n_workers)</code>  . </li><li>Alternatively, you can pass <code>n_workers=0</code> in the AzureVMClusters and later scale up the workers via <code>scale_workers</code>. In this case you won't strictly have to use <code>client.wait_for_workers(n_workers)</code>   explicitely. However having it won't create any issues since if the workers are up, the <code>client.wait_for_workers</code> will return immediately. So that is fine. </li></ol><p> </p><p>You just need one or the other. </p><p> </p><p>Note: Calling <code>scale_workers</code>  when workers are already allocated is fine. It will return back immediately. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Yeah, you're right. I wanted to include <code>scale_workers</code>  as an explicit step to point out that we need to wait on Azure to spin up the VMs and wait for them to be up and running as the documentation is not explicit on their end.</p><p> </p>

Reply:
Sounds good.